### DISCUSSION TITLE
Don't let coral die

### DISCUSSION THREAD

**Discussion by emielderckx** [PurpurMC/Purpur#1037] [__Completed__] [answered]
Let's save the turtles by adding a config option to keep coral alive when placed on land.

Possible options to specify between coral blocks, coral fans, and coral. Or even their specific variants.

Idea by: https://modrinth.com/mod/immortal-coral

**Comment by encode42** [comment, marked as answer]
Completed in d1485fa5b65ea06b26d0d4bf13c5f1eea151ab18
